README.md

```diff
@@ -14,12 +14,12 @@ You can link against this library in your program at the following coordiates:
 ```
 groupId: com.databricks
 artifactId: spark-csv_2.10
-version: 0.2.0
+version: 1.0.0
```

Comment:
Per Michael and Aaron, we will make csv and avro versions that work with spark 1.3+ start with 1.0.0